subsys/shell/Kconfig

```diff
@@ -94,6 +94,13 @@ config SHELL_PRINTF_BUFF_SIZE
 	  It is working like stdio buffering in Linux systems
 	  to limit number of peripheral access calls.
 
+config SHELL_PRINTF_AUTOFLUSH
+	bool "Indicate if the buffer should be automatically flushed"
+	default y
+	help
+	  Specify whether the shell's flash printing functions should automatically
```

Comment:
What does "flash printing functions" cover here?

Reply:
It's a mistake, I will remove it.